### DISCUSSION TITLE
What kind of filament are you using for these parts?

### DISCUSSION THREAD

**Discussion by jamesalmeida** [aderusha/DDD-Printable-Wall-Control-System#42] [Q&A] [answered]
Does anyone think there would be an issue with printing these in PLA? Just wondering if the hooks would droop from being in a garage that gets pretty warm sometimes. Sometimes in the summer it can get around 90 degrees F in my garage. 

Has anyone tried PETG also? What filaments are you all using?

**Comment by mythmon** [comment, marked as answer]
I've been printing with Prusament PETG. It's nice and sturdy, and I don't have to worry about hot garages. I'm printing most of the Gridfinity boxes in PLA though, since they don't really need the extra strength.

**Reply by mythmon** [reply]
I did some more research. Specifically I watched [this video by CNC Kitchen that compares PLA and PETG](https://www.youtube.com/watch?v=ycGDR752fT0), among others. In that video, Stefan load tested PLA and PETG, and notably did so under heated conditions. The PLA didn't have any structural problems until around 60C, which is much hotter than I expect my garage to get. Also, the PLA was more rigid like @aderusha mentioned (and therefore more brittleness).

Given all that, I'm going to switch to using PLA for most of my hooks going forward, though I think either are still fine.

**Reply by jamesalmeida** [reply]
Thanks for the info!

**Reply by akavlie** [reply]
> The PLA didn't have any structural problems until around 60C, which is much hotter than I expect my garage to get.

I put up a bunch of PLA tool hooks in a shed in Phoenix.
Nothing has fallen down, but I just checked, and a number of the tool hooks are deformed and sagging. I'll have to redo everything in PETG.

Keep this in mind if you live in a similarly hot climate.

**Comment by aderusha** [comment]
PLA is generally a little more rigid overall (right up until it breaks), PETG has a bit of yield before it fails which can be helpful and also has better heat tolerance.  Both solutions should work, I use one or the other depending on the situation (which often comes down to "what is currently in one of my printers").  PLA often wins on account of printing faster.

**Comment by psiberfunk** [comment]
ABS seems to work well, but the parts sometimes fit a little tight because of shrinkage.

**Comment by rgeyer** [comment]
I've been using PLA exclusively. Still pretty early yet, and it's still winter here so the garage is cold rather than hot. I don't anticipate any issues tho.

**Comment by psiberfunk** [comment]
Go for ASA or CF-PETG if you can.  PLA is known to creep over time.
